### PR TITLE
Merge sidechannel to develop

### PR DESCRIPTION
**Pull request by 0xAshish**

### CODE REVIEW

File: contracts/staking/stakeManager/StakeManager.sol [hunk 1]
```diff
@@ -356,7 +357,8 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
         totalStaked = totalStaked.sub(amount);
 
         NFTContract.burn(validatorId);
-        signerToValidator[validators[validatorId].signer] = INCORRECT_VALIDATOR_ID;
+        signerToValidator[validators[validatorId]
+            .signer] = INCORRECT_VALIDATOR_ID;
```
Comment (imyourm8) [inline contracts/staking/stakeManager/StakeManager.sol:361]: Can you configure the linter so it doesn't break this line? It is probably max width parameter or something.

File: test/helpers/utils.js [hunk 2]
```diff
@@ -53,6 +52,29 @@ export function getSigs(wallets, votedata) {
     .filter(d => d)
 }
 
+export function getSigsWithVotes(wallets, data, sigPrefix, maxYesVotes) {
+  wallets.sort((w1, w2) => {
```
Comment (imyourm8) [inline test/helpers/utils.js:56]: Also make a copy of the wallets, last i time i was wondering why my input array was re-ordered.

Comment (0xAshish) [reply test/helpers/utils.js:57]: done

File: contracts/staking/stakeManager/StakeManager.sol (hunk 1 above)
Comment (0xAshish) [reply contracts/staking/stakeManager/StakeManager.sol:361]: what should be the max length?

**Pull request merged**